packages/common-grpc/src/service.js

```diff
@@ -202,11 +221,14 @@ function GrpcService(config, options) {
 
   Object.keys(protoServices).forEach(function(name) {
     var protoConfig = protoServices[name];
-    var service = self.loadProtoFile_(protoConfig, config);
+    var service = GrpcService.prototype.loadProtoFile_
```

Comment:
@stephenplusplus I had to make this change to make the datastore tests work. Everywhere else util.inherit is used to extend from Service except for datastore were modelo.inherits is used (needed because it does multiple inheritance). Modelo doesn't seem to like the prototype loaded by proxyquire and self.loadProtoFile_ is not available.

It suspect it has something to do with the fact that super_ refers to only the first inherited class, modelo has a workaround [this](https://github.com/kevinconway/Modelo.js). When reviewing the change set and running the integration tests, could you pay close attention to this specific change? All the unit tests pass, this was a tricky change that makes me a little nervous considering how js interprets `this` ...

Reply:
Thanks for calling this out. It shouldn't be a big deal. `proxyquire` has its quirks, and this seems like a fair thing for it to get confused about.